sku.0/sys.server/compiled/game/script/item/tool/crafted_armor_customization_kit.java

```diff
@@ -26,14 +26,11 @@ public crafted_armor_customization_kit()
     public static final String TITLE = "@tool/customizer:armor_customize_title";
     public static final String PROMPT = "@tool/customizer:armor_customize_prompt";
     public static final int OBJECT_COLOR_MAX = 4;
+    public static final boolean OVERRIDE = getConfigSetting("GameServer", "armorColorKitOverride").equals("1");

```

Comment:
Check that getConfigSetting() doesn't result in NULL if the setting is not found.  If it does, then you'll get a null pointer issue here when this script is instantiated.